packages/r/raylib/xmake.lua

```diff
@@ -42,6 +42,10 @@ package("raylib")
     end
     add_deps("opengl", {optional = true})
 
+    add_configs("use_external_glfw", {description = "Link raylib against system GLFW instead of embedded one.", default = "OFF", values = {"OFF", "ON", "IF_POSSIBLE"}})
```

Comment:
please do not use OFF/ON, we should use `true/false`, and remove `use_` prefix

```lua
    add_configs("external_glfw", {description = "Link raylib against system GLFW instead of embedded one.", default = false, type = "boolean"})
```

and it is enabled, you need add `add_deps("glfw")`

Reply:
Ah got it. I was following how it was mentioned in raylib wiki. Will fix these.

Reply:
I have now changed the file according to feedback. Did I understand correctly?

Also does the `    on_install("windows", "linux", "macosx|arm64", "mingw", function (package)` need to have also `android` `web` and `raspberry-pi` ?

Reply:
Seems that if I set `-DCUSTOMIZE_BUILD=ON` then none of my builds actually run. I think I'm missing something.

I will look into this more, will keep you updated. 

Of course if you or anyone else who sees this has something proper working already I'd appreciate any help.

Reply:
Okay, I had approached this in completely wrong way. I removed the custom opengl and glfw setting.

Instead, I'm checking now the architecture and platform, then setting `-DPLATFORM` correctly depending on it.

Please check and tell me if there's anything I could've done better :)

Reply:
Well, it works on my linux pc but not on my raspberry pi.

I don't know what to do :(

Reply:
> I have now changed the file according to feedback. Did I understand correctly?
> 
> Also does the ` on_install("windows", "linux", "macosx|arm64", "mingw", function (package)` need to have also `android` `web` and `raspberry-pi` ?

they are only raylib, not for xmake. you can see platform list of xmake. `xmake f -h`

Reply:
> Well, it works on my linux pc but not on my raspberry pi.
> 
> I don't know what to do :(

raspberry pi is also linux platform in xmake. You will need to fix it based on the actual compilation error.

Reply:
or you should open an issue in raylib to fix your errors on raspberry pi. or use `add_patches` to fix it.

Reply:
I'll remake whole PR at some point when i have more time

Thank you for being patient with me